### ISSUE TITLE
[BUG] Renamed files return original file name instead of new file name

### ISSUE THREAD

**Issue by atishaninov-xometry** [tj-actions/changed-files#1280]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

Between versions v36.1.0 and v36.2.0 the behaviour for renamed files are changed. For cases where the action is triggered from PR in which the file has been renamed, previously the action returned the current file name after renaming. After version v36.2.0 and until now, this action returns original file name before renaming.

I use this action to run linters on modified files, and this change in behavior has started to produce "file does not exist" errors.
I couldn't find any corresponding changelog entries or inputs to adjust this behaviour to old style.

### To Reproduce

```yaml
on:
  pull_request:
    types: [opened, reopened, synchronize]

jobs:
  diff:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
        with:
          fetch-depth: 2

      - id: changed-files-36-1-0
        uses: tj-actions/changed-files@v36.1.0
      - run: echo ${{ steps.changed-files-36-1-0.outputs.all_changed_files }}

      - id: changed-files-36-2-0
        uses: tj-actions/changed-files@v36.2.0
      - run: echo ${{ steps.changed-files-36-2-0.outputs.all_changed_files }}
```
1. Create new empty repo with specified action and one file `example_file.txt` and arbitrary content
2. Create new branch, rename this file to `renamed_example_file.txt`
3. Submit PR

### What OS are you seeing the problem on?

ubuntu-latest or ubuntu-20.04

### Expected behavior?

Outputs 
`${{ steps.changed-files-36-1-0.outputs.all_changed_files }}`
`${{ steps.changed-files-36-2-0.outputs.all_changed_files }}`
should be the same and equals to `renamed_example_file.txt`

### Relevant log output

```shell
Debug log into comments
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by atishaninov-xometry** [comment]
[logs_2.zip](https://github.com/tj-actions/changed-files/files/11790667/logs_2.zip)

**Comment by jackton1** [comment]
@atishaninov-xometry I'll suggest using the latest version [v36.4.0](https://github.com/tj-actions/changed-files/releases/tag/v36.4.0) to replicate the issue.

**Comment by jackton1** [comment]
I confirmed that a bug was introduced in v36.2.0. I create a fix for this.

**Comment by toddsdarling** [comment]
Is there going to be an update to the `tj-actions/eslint-changed-files` action which uses this action?

**Comment by toddsdarling** [comment]
> Is there also going to be an update to the `tj-actions/eslint-changed-files` action which uses this action?

Oh, nevermind....I just realized that action automatically pulls in the latest version of this action.  So once a release gets published it should fix.  Thank you!

**Comment by jackton1** [comment]
@toddsdarling @atishaninov-xometry The fix should be available in the latest release

**Comment by atishaninov-xometry** [comment]
@jackton1 i can confirm that issue is fixed in v36.4.1 and now working as expected. Thank you for fast fix!
